cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -435,16 +436,49 @@ private static <I extends ReactiveLimitsHolder & Injection<I>> void writeSynchro
         SynchronousMachineEq.write(context.getNamingStrategy().getCgmesId(i), i.getNameOrId(),
                 context.getNamingStrategy().getCgmesId(i.getTerminal().getVoltageLevel()),
                 generatingUnit, regulatingControlId, reactiveLimitsId, minQ, maxQ, ratedS, defaultRatedS, kind, cimNamespace, writer, context);
+        String hydroPlantStorageType = i.getProperty(Conversion.PROPERTY_CGMES_SYNCHRONOUS_MACHINE_HYDRO_PLANT_STORAGE_KIND);
+        String hydroPowerPlantId = null;
+        if (hydroPlantStorageType != null && GeneratingUnitEq.generatingUnitClassName(energySource).equals("HydroGeneratingUnit")) {
+            String hydroPowerPlantName = i.getNameOrId();
+            hydroPowerPlantId = context.getNamingStrategy().getCgmesId(refTyped(i), HYDRO_POWER_PLANT);
```

Comment:
Here for me it would be better to use `ref(i)`, because I need the seed for the Hydro power plants not to include "_SM".

Reply:
Done